step-automation-packages/step-automation-packages-junit/src/main/java/step/engine/plugins/AutomationPackageKeywordsPlugin.java

```diff
@@ -50,6 +50,7 @@ public void initializeExecutionEngineContext(AbstractExecutionEngineContext pare
                                     new ObjectId(),
                                     context.require(FunctionTypeRegistry.class),
                                     functionAccessor,
+                                    resourceManager,
```

Comment:
@david-stephan 
I forgot again, what the manager is used in execution engine context. I suppose, that in 100% this manager is prepared 
in `AbstractStepContext#setDefaultAttributes` (i.e `new LocalResourceManagerImpl(new File(contextPath() + "/resources")...`). 

So we have a guarantee, that the manager for isolated AP is always local?

Reply:
@iegorov777 the resource manager for execution context can be overriden, it is the case for executions in controller mode, here the plugin is only used in context of local execution, so this should be safe